xml/Microsoft.Extensions.ObjectPool/DefaultPooledObjectPolicy`1.xml

```diff
@@ -138,9 +138,10 @@
         <Parameter Name="obj" Type="T" />
       </Parameters>
       <Docs>
-        <param name="obj">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="obj">The object to return to the pool.</param>
+        <summary>Runs some processing when an object was returned to the pool. Can be used to reset the state of an object and indicate if the object should be returned to the pool.</summary>
+        <returns>
+          <see langword="true" /> if the object should be returned to the pool. <see langword="false" /> if it's not possible/desirable for the pool to keep the object.</returns>
```

Comment:
I realize this is what the existing comment says, but should this say:

```suggestion
          <see langword="true" /> if the object should be returned to the pool. <see langword="false" /> if it's not possible or desirable for the pool to keep the object.</returns>
```

Similarly elsewhere

Reply:
Thank you @pranavkm for providing a suggestion. It's ok to change the original texts from triple slash, since the purpose of the dotnet-api-docs PRs is to get them through language review, like you did here 😄 . I'll apply your suggestion in all the places where I find similar text.

Reply:
Thank you @pranavkm for providing a suggestion. It's ok to change the original texts from triple slash, since the purpose of the dotnet-api-docs PRs is to get them through language review, like you did here 😄 . I'll apply your suggestion in all the places where I find similar text.